rest-api/test/unit_test/offline_test/biobank_samples_pipeline_mysql_test.py

```diff
@@ -166,9 +166,46 @@ def test_reconciliation_query(self):
     self._insert_samples(p_on_time, BIOBANK_TESTS[:2], ['GoodSample1', 'GoodSample2'], 'OGoodOrder',
                          within_24_hours, within_24_hours - datetime.timedelta(hours=1))
 
+    # On time, recent order and samples not confirmed do not show up in reports.
+    p_unconfirmed_samples = self._insert_participant()
+    self._insert_order(p_unconfirmed_samples, 'not_confirmed_order', BIOBANK_TESTS[:4], order_time,
+                       finalized_tests=BIOBANK_TESTS[:3], kit_id='kit3', tracking_number='t3',
+                       collected_note=u'\u2013foo', processed_note='bar', finalized_note='baz')
+    self._insert_samples(p_unconfirmed_samples, BIOBANK_TESTS[:3], ['Unconfirmed_sample',
+                         'Unconfirmed_sample2'], 'Ounconfirmed_sample', None, within_24_hours)
+
+    # two day old order not confirmed shows up in missing.
```

Comment:
You should assert that the tests from this order specifically show up below.

Reply:
done